scripting/2.0/modules/ROOT/pages/scripting-module-studio.adoc

```diff
@@ -0,0 +1,237 @@
+
+* In the *Required Libraries* section, click *Configure...* and select one of the following options to search and install the engine: +
+
+** Add recommended libraries +
+Install Groovy engine as recommended. This option only appears if you did not previously install Groovy.
+** Use local file +
+Browse a local file for the required engine library and install it.
+** Add Maven dependency +
+Browse the engine dependency and install it. +
+
+* In the `pom.xml` file, manually add the engine in both of these sections:
+** `<sharedLibraries>` tags under the `<configuration>` section
+** `<dependency>` tags under the `<dependencies>` section
+
+[start=5]
+. In the *General* section of the *Execute* tab, click the refresh button to update the engine values list for the operation.
+. In *Engine*, select the installed engine.
+. Provide your script code in the *Code* text box. +
+Additionally, you can load script code from an external file by using a xref:mule-runtime::configuring-properties.adoc#_file_properties[file configuration property].
+
+In Studio, configuring Jython engine looks as follows:
+
+.Installing engine using Configure button
+image::scripting-configure-button.png[Installing engine using Configure button]
+
+In the `pom.xml` file, manually adding Jython engine looks as follows:
+
+[source,xml,linenums]
+----
+<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-clean-plugin</artifactId>
+				<version>3.0.0</version>
+			</plugin>
+			<plugin>
+				<groupId>org.mule.tools.maven</groupId>
+				<artifactId>mule-maven-plugin</artifactId>
+				<version>${mule.maven.plugin.version}</version>
+				<extensions>true</extensions>
+				<configuration>
+				<sharedLibraries>
+                        <sharedLibrary>
+                            <groupId>org.python</groupId>
+                            <artifactId>jython-standalone</artifactId>
+                        </sharedLibrary>
+        </sharedLibraries>
+                </configuration>
+			</plugin>
+		</plugins>
+	</build>
+
+<dependencies>
+  <dependency>
+              <groupId>org.python</groupId>
+              <artifactId>jython-standalone</artifactId>
+              <version>2.7.2</version>
+  </dependency>
+</dependencies>
+----
+
+In Studio, the Scripting Module configuration with script code looks as follows:
+
+.Scripting Module configuration with script code
+image::scripting-studio-flow.png[Scripting Module configuration with script code]
+
+In the *Configuration XML* editor, the XML looks as follows:
+
+[source,xml,linenums]
+----
+<scripting:execute engine="python" doc:name="Script">
+    <scripting:code>
+      def factorial(n):
+          if n == 0: return 1
+	  return n * factorial(n-1)
+
+      result = factorial(10)
+    </scripting:code>
+</scripting:execute>
+----
+
+In Studio, the loaded script code with a file configuration property looks as follows:
+
+.Script code with a File Configuration Property
```

Comment:
Reword or add a comment below to make it more googleable.

"How to use a external file on scripting module" something like that, "File Configuration Property" is something that nobody knows or understand